pkg/components/ytsaurus_client.go

```diff
@@ -518,8 +519,12 @@ func (yc *ytsaurusClient) doSync(ctx context.Context, dry bool) (ComponentStatus
 	if yc.ytClient == nil {
 		token, _ := yc.secret.GetValue(consts.TokenSecretKey)
 		timeout := time.Second * 10
+		proxy, ok := os.LookupEnv("YTOP_PROXY")
+		if !ok {
+			proxy = yc.cfgen.GetHTTPProxiesAddress(consts.DefaultHTTPProxyRole)
+		}
 		yc.ytClient, err = ythttp.NewClient(&yt.Config{
-			Proxy:               yc.cfgen.GetHTTPProxiesAddress(consts.DefaultHTTPProxyRole),
+			Proxy:               proxy,
```

Comment:
Don't you want to enable DisableProxyDiscovery by default? Looks like it works unless some heavy commands are added into initialization pipeline.

Reply:
Why won't we enable disable, indeed.